config/examples/plan_rules.yml

```diff
@@ -38,7 +38,6 @@ plan_rules: &default
       - account_plans
       - multiple_users
       - groups
-      - end_users
```

Comment:
Right :+1: But this has some implications:
1. Everyone in the team right after merging will have to do `yes | cp -rf config/examples/plan_rules.yml config/plan_rules.yml` or they environment will fail sooner or later (perhaps when resetting the DB).
2. The most important thing now is removing the references to `end_users` also in [the config/plan_rules.yml that we use for production (and preview) in SaaS](https://github.com/3scale/deploy/blob/master/system/data/production/plan_rules.yml), or we will get bad errors.
3. I do not know if the on-prem customers know about this config file, but if so, we should add documentation for upgrading to the next on-prem version saying that they should make this change in their config/plan_rules.yml

Reply:
Question: isn't the config supposed to be copied from config/example each time?

Reply:
I think end-users and end-user-plans were never activated for on-prem but somebody needs to check.